doc/ledger3.texi

```diff
@@ -5770,27 +5770,31 @@ Specify the location of the init file. The default is @file{~/.ledgerrc}.
 Display the options in effect for this Ledger invocation, along with
 their values and the source of those values, for example:
 
-@smallexample @c command:A9349E4,with_input:03ACB97
-$ ledger --options bal --cleared
-@end smallexample
-
-@smallexample @c output:A9349E4
-===============================================================================
+@smallexample
+$ ledger --options bal --cleared -f ~/ledger/test/input/drewr3.dat
```

Comment:
On 12.01.15,12:10, Alexis Hildebrandt wrote:
> > @@ -5770,27 +5770,31 @@ Specify the location of the init file. The default is @file{~/.ledgerrc}.
> >  Display the options in effect for this Ledger invocation, along with
> >  their values and the source of those values, for example:
> >  
> > -@smallexample @c command:A9349E4,with_input:03ACB97
> > -$ ledger --options bal --cleared
> > -@end smallexample
> > -
> > -@smallexample @c output:A9349E4
> > -===============================================================================
> > +@smallexample
> > +$ ledger --options bal --cleared -f ~/ledger/test/input/drewr3.dat
> 
> @jostber How would this change improve the documentation?
> 

I am not sure what happened here. I think I'll close this one. 

Jostein